service/league_service/league_service.py

```diff
@@ -92,8 +92,8 @@ async def update_data(self):
                     [
                         LeagueDivision(
                             row[league_season_division.c.id],
-                            row[league_season_division.c.from_mean],
-                            row[league_season_division.c.to_mean],
+                            row[league_season_division.c.min_rating],
+                            row[league_season_division.c.max_rating],
```

Comment:
Renaming these needs some refactoring further down in the python model and then also in the database to keep the names synchronized.
You can either rename them in `service/db/models.py` and the migration file, too, or I can do it if you want me to.
(This might be the main reason the unit tests are currently failing.)

Reply:
Done, tests are passing again